www/include/configuration/configServers/popup/popup.php

```diff
@@ -25,9 +25,28 @@
 require_once _CENTREON_PATH_ . 'bootstrap.php';
 require_once _CENTREON_PATH_ . "www/include/common/common-Func.php";
 require_once _CENTREON_PATH_ . "/www/class/centreonRestHttp.class.php";
-
+require_once _CENTREON_PATH_ . "/www/class/centreonSession.class.php";
 $pearDB = $dependencyInjector['configuration_db'];
 
+/* Check Session */
+CentreonSession::start(1);
+if (!CentreonSession::checkSession(session_id(), $pearDB)) {
+    print "Bad Session";
+    exit();
+}
+$centreon = $_SESSION['centreon'];
+$pollerId = filter_var($_GET['id'] ?? null, FILTER_VALIDATE_INT);
+$userId = $centreon->user->user_id;
+$isAdmin = $centreon->user->admin;
+
+$acl = new CentreonACL($userId, $isAdmin);
+$aclPollers = $acl->getPollers();
+
+if (!array_key_exists($pollerId, $aclPollers)) {
```

Comment:
```suggestion
if ($pollerId === false || !array_key_exists($pollerId, $aclPollers)) {
```

Reply:
to avoid this : WARNING array_key_exists(): The first argument should be either a string or an integer

Reply:
already pushed ;)